kafl_fuzzer/worker/qemu.py

```diff
@@ -417,7 +417,7 @@ def send_payload(self):
             sys.exit(0)
 
         # for -R > 1, count and toggle reload_mode at runtime
-        if self.config.reload > 1:
+        if self.config['reload'] > 1:
```

Comment:
is this correct?

Reply:
or, can we rename it without affecting the user-visible config/argparse input?

Reply:
Renaming a user config option because our we have a conflict with the configuration management tool we are using, while also having a workaround thanks to the extra dict notation, is unecessary in my opinion.

However, using that PR to update obscure config option with better names is a good opportunity.

Reply:
my question was, can you rename the variable without affecting the user-visible config option.
I know argparse can do it, and I guess dynaconf as well?

Reply:
Yes, the `Validator` can accept a list of `*names`:
![image](https://user-images.githubusercontent.com/964610/203997574-60777280-2805-4dd6-ae23-f8b8c7e0eaa0.png)

How do you do that with `argparse` ?

Reply:
With dest= option:
https://docs.python.org/3/library/argparse.html#dest
https://github.com/intel/ccc-linux-guest-hardening/blob/8b1f1588bba37a23c020769fe8854e77c48a1f3b/bkc/kafl/init_harness.py#L314

Reply:
Fixed.